### PR TITLE
my codes so far

### PR DESCRIPTION
**Pull request by Hamidreza-Fakharzadeh**
please could you instruct me i have some issues about passing the tests

### CODE REVIEW

File: week-1/Homework/mandatory/1-syntax-errors.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+};
 
+function introduceMe(name, age) {
+return `Hello, my name is ${name} and I am ${age} years old`;
+};
 function getRemainder(a, b) {
-    remainder = a %% b;
+    remainder = a % b;
```
Comment (zubhav) [inline week-1/Homework/mandatory/1-syntax-errors.js]: Good! Always declare your variables as `const` or `let`. It is good practice to do this and you will soon understand the value of it once we get into more advanced topics.

File: week-1/Homework/mandatory/1-syntax-errors.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+function addNumbers(a, b, c) {
     return a + b + c;
-}
-
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+};
 
+function introduceMe(name, age) {
+return `Hello, my name is ${name} and I am ${age} years old`;
```
Comment (zubhav) [inline week-1/Homework/mandatory/1-syntax-errors.js]: Great. To format this nicely, you can indent the code inside the function using tab. By doing so, we can see where the content of the function is located clearly.

File: week-1/Homework/mandatory/2-logic-error.js [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 function getWordLength(word) {
-  return "word".length()
+  return word.length;
 }
 
 function multiply(a, b, c) {
-  a * b * c;
-  return;
+  
+  return a * b * c;;
```
Comment (zubhav) [inline week-1/Homework/mandatory/2-logic-error.js]: Good! You can remove the extra semi-colon at the end of this line too. It is not doing anything extra :)

File: week-1/Homework/mandatory/3-function-output.js [hunk 4]
```diff
@@ -1,16 +1,21 @@
 // Add comments to explain what this function does. You're meant to use Google!
+//generate umber between 0 and 9
```
Comment (zubhav) [inline week-1/Homework/mandatory/3-function-output.js]: A question to think about...

What `type` of value does this return? A `number` or a `float`? How can we be sure of it?

If it is a `float`, how can we turn this into a `number`? Think about the capabilities of the `Math` object and what we learnt in the last lesson.

File: week-1/Homework/mandatory/3-function-output.js [hunk 5]
```diff
@@ -1,16 +1,21 @@
...
 function getNumber() {
     return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+//  The concat() method concatenates the string arguments to the calling string and returns a new string.
```
Comment (zubhav) [inline week-1/Homework/mandatory/3-function-output.js]: 👍

File: week-1/Homework/mandatory/3-function-output.js [hunk 6]
```diff
@@ -1,16 +1,21 @@
...
+
+
 function s(w1, w2) {
     return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
     // Write the body of this function to concatenate three words together
     // Look at the test case below to understand what to expect in return 
+    return firstWord.concat(secondWord,thirdWord);
```
Comment (zubhav) [inline week-1/Homework/mandatory/3-function-output.js]: Nice

File: week-1/Homework/mandatory/4-tax.js [hunk 7]
```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(productPrice) {
+  return productPrice + 0.2*productPrice;
```
Comment (zubhav) [inline week-1/Homework/mandatory/4-tax.js]: Great. Well done. Another formatting tip for consistency, use spacing between your operators and variables where you can. `return productPrice + 0.2 * productPrice;`

File: week-1/Homework/mandatory/4-tax.js [hunk 8]
```diff
@@ -17,7 +19,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+function formatCurrency(productPrice) {
+  calculateSalesTax(productPrice)
```
Comment (zubhav) [inline week-1/Homework/mandatory/4-tax.js]: If the input of this function is 11:

Your function calls `calculateSalesTax` and returns the following
11 + (0.2 * 11) = **13.2**

The business has asked us to provide 2 decimal places and £0.00 format, so your function should return £13.20. Think about how you could implement this.

**Comment by Hamidreza-Fakharzadeh** [comment]
i  have passed all the tests in magic ball 8( task)

**Pull request closed without merging**